pages/vision.tsx

```diff
@@ -0,0 +1,92 @@
+        <section className={styles.Section6}>
+          <div className={styles.left}>
+            <div className={styles.Section6Text}>
+              <h2 className={styles.green}>VISION</h2>
+            </div>
+            <div className={styles.Section6Text}>
+              <h3 className={styles.space}>The cornerstone of the internet of tomorrow</h3>
+            </div>
+            <div className={styles.Section6Grid}>
+              <p className={styles.space}>Our ambition is to establish a decentralized communication protocol as a cornerstone of the internet of tomorrow. We aim to empower web3 developers to bring about unstoppable peer-to-peer communication to billions of devices. </p>
+            </div>
+            <div className={styles.Section6Grid}>
+
+              <p className={styles.space}>We are driven by a mission to empower humanity with the power of unrestricted communication, free from the limitations of centralized infrastructure and censorship.</p>
+            </div>
+            <div className={styles.Section6Grid}>
+
+              <p className={styles.space}>Our open-source approach promotes contributions and transparency, allowing anyone to take part in our pursuit of uncompromised communication.</p>
+            </div>
+
+          </div>
+          <div className={styles.right}>
+            <img src="./img/imgSection6.png" alt="" />
+          </div>
+        </section>
+
+        <section className={styles.Section6}>
+          <div className={styles.left}>
+            <div className={styles.Section6Text}>
+              <h3 className={styles.space} >Crossing the chasm</h3>
+            </div>
+            <div className={styles.Section6Grid}>
+
+              <p className={styles.space}>Wesh Network is poised to become a key component of the alternate Internet, powering numerous decentralized systems such as blockchains, messaging apps, dApps, and social tools. As such, our toolkit will become a fundamental component of the Web3 builder&apos;s workshop.</p>
+            </div>
+            <div className={styles.Section6Grid}>
+
+              <p className={styles.space}>Through collaborations and integrations with other related projects that follow a decentralized approach, our protocol will reach new heights of capability and performance. We&apos;ve partnered with governments, innovators in the web3 space, or military forces, who plan to implement Wesh Protocol to build secure and resilient communication tools for extreme contexts.</p>
+            </div>
+            <div className={styles.Section6Grid}>
+
+              <p className={styles.space}>The Wesh Network protocol is designed to offer unparalleled resilience coupled with interoperability, security, and modularity. Humanity deserves access to free, reliable and censorship-resistant communication. Berty is building the Wesh Network to fulfill this mission and become the foundation of unstoppable communication for the Internet&apos;s next era.</p>
```

Comment:
"unparalleled resilience".  The word "resilient" was just used in the previous sentence. It sounds repetitious. Maybe change the first "resilient communication" to "robust communication"?